app/code/BodenkoVV/AskQuestion/Observer/Catalog/Block/Product/View.php

```diff
@@ -0,0 +1,44 @@
+<?php
+namespace PalashAs\AskQuestion\Observer\Catalog\Block\Product;
+
+use Magento\Framework\Event\Observer;
+use Magento\Framework\Registry;
+
+/**
+ * Class View
+ * @package PalashAs\AskQuestion\Observer\Catalog\Block\Product
+ */
+class View implements \Magento\Framework\Event\ObserverInterface
+{
+    /** Product Ask Question Custom Layout Name */
+    const LAYOUT_HANDLE_NAME = 'catalog_product_view_askquestiontab';
+
+    /** @var Registry */
+    protected $_registry;
+
+    /**
+     * View constructor.
+     * @param Registry $registry
+     */
+    public function __construct(Registry $registry)
+    {
+        $this->_registry = $registry;
+    }
+
+    /**
+     * @param Observer $observer
+     * @return $this
+     */
+    public function execute(Observer $observer)
+    {
+        $event = $observer->getEvent();
```

Comment:
Was unable to found where this observer is attached to any of frontend event. Currently the form is shown on product page regardless if the attribute allow_to_ask_questions has value "yes" or "no".Please re-check this event and its observer

Reply:
+